support-frontend/app/controllers/CanonicalLinks.scala

```diff
@@ -1,17 +0,0 @@
-package controllers
-
-trait CanonicalLinks {
```

Comment:
This file has actually been renamed to `RegionalisedLinks` to make it clearer what its purpose is because it is now just providing regionalised versions of urls for redirects - not the canonical link provided to search engines.